plugins/SegmentEditor/templates/_segmentSelector.twig

```diff
@@ -56,10 +56,10 @@
                     </strong></div>
                 {{ 'General_And'|translate }} <div class="auto_archive"><strong>
                         <select class="auto_archive_select">
-                            {% if createRealTimeSegmentsIsEnabled %}
+                            {% if createRealTimeSegmentsIsEnabled or isBrowserArchivingEnabled %}
                             <option selected="1" value="0">{{ 'SegmentEditor_AutoArchiveRealTime'|translate }} {{ 'General_DefaultAppended'|translate }}</option>
                             {% endif %}
-                            <option {% if not createRealTimeSegmentsIsEnabled %}selected="1"{% endif %} value="1">{{ 'SegmentEditor_AutoArchivePreProcessed'|translate }} </option>
+                            <option {% if not createRealTimeSegmentsIsEnabled and not isBrowserArchivingEnabled %}selected="1"{% endif %} value="1">{{ 'SegmentEditor_AutoArchivePreProcessed'|translate }} </option>
```

Comment:
btw probably not an issue but do we maybe need to remove `selected="1"` from the previous option when selecting this one by default? I suppose the browser in this case automatically selects the later one?

I'm thinking maybe it needs to use an `or` and not an `and`? Like we want to preselect this one either when browser archiving is disabled or when real time segment is not allowed?

Reply:
They should be mutually exclusive conditions.

Reply:
@diosmosis I just disabled browser archiving but real time was still the default. It could alternatively be simplified to `if not isBrowserArchivingEnabled`. 

Otherwise it would be only the default if someone was to disable real time segments but only few people would do this.

Reply:
@tsteur that may be because browser archiving of segments is still enabled. Can you check that?

Reply:
That's possible. I think we'd want to still enable it though by default in that case see https://github.com/matomo-org/matomo/issues/6210#issuecomment-712408748
![image](https://user-images.githubusercontent.com/273120/97935268-f37d0700-1ddc-11eb-8777-dffce31e7f0d.png)

and then in https://github.com/matomo-org/matomo/issues/6210#issuecomment-712464414
![image](https://user-images.githubusercontent.com/273120/97935308-0ee81200-1ddd-11eb-9b5b-7b0336f797a9.png)

It wasn't really meant in relation to the segment specific archive setting but just the general browser archiving setting.

Reply:
Ok, I will ignore segment specific archiving.

Reply:
@tsteur changed.